### ISSUE TITLE
resend double opt in email

### ISSUE THREAD

**Issue by marcinkoziej** [fixthestatusquo/proca-server#24]

**Comment by marcinkoziej** [comment]
@tttp Your issue - what do you have in mind ? Resend after N hours ?

**Comment by tttp** [comment]
Yeah, v2 idea for improvement: sending a reminder "re: confirm your action" if the status is still pending after x hours.
can we parc this one?
somewhat related: do you handle bounces?
